cloudfoundry/managers/session.go

```diff
@@ -248,7 +311,10 @@ func (s *Session) init(config *configv3.Config, configUaa *configv3.Config, conf
 
 		var accessTokenSess string
 		var refreshTokenSess string
-		if configUaa.UAAOAuthClient() == "cf" {
+		if configSess.SSOPasscode != "" || loadedTokens {
+			accessTokenSess = accessToken
+			refreshTokenSess = refreshToken
+		} else if configUaa.UAAOAuthClient() == "cf" {
```

Comment:
this part should be revert as passcode is a cloud foundry user and not an uaa user